modules/cf-core/src/methods/state-channel/deploy-state-deposit-holder/controller.ts

```diff
@@ -21,8 +24,8 @@ import {
   NO_TRANSACTION_HASH_FOR_MULTISIG_DEPLOYMENT
 } from "../../errors";
 
-// Estimate based on:
-// https://rinkeby.etherscan.io/tx/0xaac429aac389b6fccc7702c8ad5415248a5add8e8e01a09a42c4ed9733086bec
+// Estimate based on rinkeby transaction:
+// 0xaac429aac389b6fccc7702c8ad5415248a5add8e8e01a09a42c4ed9733086bec
```

Comment:
why not leave in the full link?

Reply:
linter was complaining about line being too long
I'm a slave to the linter